### PR TITLE
feat: add subscribeKeys function

### LINKED ISSUE #262: Missing `subscribeKeys(store, keys, cb)` method ?
We have both `store.subscribe(cb)` and `store.listen(cb)` to decide if we want to call the listeners immediately on subscribe or not.

So why not having the same with `subscribeKeys(store, keys, cb)` along `listenKeys(store, keys, cb)` ?

```typescript
import { listenKeys, type StoreValue } from 'nanostores';

export function subscribeKeys<
  SomeStore extends { get: () => StoreValue<SomeStore>; setKey: (key: any, value: any) => void }
>(
  $store: SomeStore,
  propsToListen: SomeStore extends { setKey: (key: infer Key, value: never) => unknown } ? readonly Key[] : never,
  callback: (_props: StoreValue<SomeStore>) => void
) {
  listenKeys($store, propsToListen, callback);
  callback($store.get());
}
```

I'm using it in prod and it's working pretty good !

### PR DESCRIPTION
**Pull request by gismya**
Adds a subscribeKeys function. It ensures there's a `subscribe` function for every `listen` function, aligning the API for all types of store listeners. Also updated the readme to describe the listenKeys function

Fixes #262

### CODE REVIEW

File: listen-keys/index.d.ts [hunk 1]
```diff
@@ -32,3 +36,40 @@ export function listenKeys<
...
+ *     console.log('You has no access')
+ *   }
+ * })
+ * ```
+ *
+ * @param $store The store to listen.
+ * @param keys The keys to listen.
+ * @param listener Standard listener.
+ */
+
```
Comment (ai) [inline listen-keys/index.d.ts:59]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ A tiny state manager for **React**, **React Native**, **Preact**, **Vue**,
 **Svelte**, **Solid**, **Lit**, **Angular**, and vanilla JS.
 It uses **many atomic stores** and direct manipulation.
 
-* **Small.** Between 286 and 818 bytes (minified and brotlied).
+- **Small.** Between 287 and 818 bytes (minified and brotlied).
```
Comment (ai) [inline README.md:10]: Can you revert code style changes in docs?

Comment (gismya) [reply README.md:10]: Yes. Was careful to avoid it until I did a search and replace for 286 -> 287 :)

**Comment by ai** [comment]
Thanks. I will merge it on next minor release.

**Pull request merged**